python/pyarrow/_parquet.pyx

```diff
@@ -1059,25 +1059,34 @@ cdef class ParquetReader:
 
     def read_row_group(self, int i, column_indices=None,
                        bint use_threads=True):
+        return self.read_row_groups([i], column_indices, use_threads)
+
+    def read_row_groups(self, row_groups not None, column_indices=None,
+                        bint use_threads=True):
         cdef:
             shared_ptr[CTable] ctable
+            vector[int] c_row_groups
             vector[int] c_column_indices
 
         if use_threads:
             self.set_use_threads(use_threads)
 
+        for row_group in row_groups:
+            c_row_groups.push_back(row_group)
+
         if column_indices is not None:
             for index in column_indices:
                 c_column_indices.push_back(index)
 
             with nogil:
                 check_status(self.reader.get()
-                             .ReadRowGroup(i, c_column_indices, &ctable))
+                             .ReadRowGroups(c_row_groups, c_column_indices, 
```

Comment:
/home/travis/build/apache/arrow/python/pyarrow/_parquet.pyx:1083:76: W291 trailing whitespace
```suggestion
                             .ReadRowGroups(c_row_groups, c_column_indices,
```

Reply:
@xhochy Thanks. I finally took the time to read the project docs which explain how to flake8 cython...